pytest_catchlog.py

```diff
@@ -71,15 +71,22 @@ def catching_logs(handler, filter=None, formatter=None,
                 yield handler
 
 
-def add_option_ini(parser, option, dest, default=None, help=None):
+def add_option_ini(parser, option, dest, action=None, default=None, help=None):
     parser.addini(dest, default=default,
                   help='default value for ' + option)
-    parser.getgroup('catchlog').addoption(option, dest=dest, help=help)
+    parser.getgroup('catchlog').addoption(option,
+                                          action=action,
+                                          dest=dest,
+                                          default=default,
```

Comment:
The tests pass... So I'm assuming it doesn't.

I think that on another PR of mine I've set default values directly when adding the option, and I probably shouldn't, defaults should be left at `None` and we pass the default value when getting the option with `get_option_ini` so both the CLI options and the INI options are looked at...